packages/text-editor/src/components/InlineCommandsList.svelte

```diff
@@ -33,6 +35,15 @@
   let dummyPopup: PopupResult
   let menuPopup: SelectPopup
 
+  let fuse: Fuse<any>
+  let sortedItems: any[] = []
+  $: fuse = new Fuse(items, { shouldSort: true, keys: ['id'] })
```

Comment:
can we get rid of dependency on fuse.js and just check that item name contains or starts with query string?